.gitmodules

```diff
@@ -4,6 +4,3 @@
 [submodule "contracts/lib/forge-std"]
 	path = contracts/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
-[submodule "contracts/lib/eigenlayer-contracts"]
-	path = contracts/lib/eigenlayer-contracts
-	url = https://github.com/Layr-Labs/eigenlayer-contracts
```

Comment:
Note that this has been removed as we can use the eigenlayer-contracts from eigenlayer-middleware dependences